src/controller/sms/steps.js

```diff
@@ -0,0 +1,61 @@
+/* eslint-disable no-param-reassign */
+const message = require('./message')
+const saveUser = require('../../user/save_user')
+const debug = require('debug')('sms')
+
+// list of steps for the SMS flow. each step is an object with a before handler,
```

Comment:
Brilliant. I love how you simplified this config. This'll make it a lot cleaner to update as we figure out the exactly messaging and flow--and later, we could have multiple steps depending on your keyword, and just switch out which set of steps you get. Really nice work.